CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Change Log
 ===
 
+## 3.1.0-rc-9
```

Comment:
3.1.1 even already released....

Reply:
Updated changelog

Reply:
Are the socket based test failures because of these changes ?

Reply:
no, released version passing unix/tcp tests (beside 2 configurations). This PR fails because amd64 native libs not found something

Reply:
I see this error 
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.newsclub.net.unix.NativeUnixSocket

But I believe this class should be available